dataCollection/settings.py.template

```diff
@@ -3,7 +3,7 @@
 import os
 PROJECT_PATH = os.path.realpath(os.path.dirname(__file__))
 
-DEBUG = True
+DEBUG = False
```

Comment:
I can get behind that. Let's add a point to the readme. We can also add a code comment to this file with that.